mne_bids/tests/test_mne_bids.py

```diff
@@ -359,6 +374,15 @@ def test_set():
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
 
+    # check events.tsv is written
+    # XXX: only from 0.18 onwards because events_from_annotations
+    # is broken for earlier versions
```

Comment:
does this mean that the bids-validator passes even if `events.tsv` is not written?

Reply:
yes, because `events.tsv`. I checked and it seems it is an optional file according to BEP008 document. However, the specification appears to be ambiguous about it. What does BEP006 say about these files? I think it should be optional since for resting state data you won't have task events.

Reply:
I just checked and yes, `events.tsv` is optional everywhere :-)